bin/dockerfilelint

```diff
@@ -34,6 +38,8 @@ if (argv._.length === 0 || argv._[0] === '-') {
   });
 }
 
+setReporter();
```

Comment:
Most definitely. I originally had it as a ternary as well but figured y'all might want to add more Reporters in the future. Now, Im realizing that there probably aren't very many versions you would want.